ruby/lib/clone_handler.rb

```diff
@@ -0,0 +1,45 @@
+require 'json'
+require 'net/http'
+
+class CloneHandler
+  def initialize(username)
+    @username = username
+  end
+
+  def call
+    return unless clonable?
+
+    Kernel.system(clone_command)
+  end
```

Comment:
is `call` a pattern common in the ruby community? It seems something more descriptive could be helpful long term. The call here is going to hit the api if possible and get all the repos.

So maybe it could be renamed `clone_all_repos` ?

Reply:
`call` is `clone` now. This clones all the repos.